win32ss/user/user32_vista/user32_vista.spec

```
@@ -0,0 +1 @@
+@ stdcall -version=0x605+ GetDpiForWindow(ptr)
```

No need for `0x605` in ReactOS' own code.
```suggestion
@ stdcall -version=0xA00+ GetDpiForWindow(ptr)
```

---

I'm moving the version gated call into user32 with user32_vista.dll not gating this call by version since it'll be required for common controls. However, we should match Windows here so I think `0x605+` is the correct way to go. If someone runs components written for Windows 10 technical preview it'll work on Windows but not ReactOS if we did it your way.